VariantEncoding.md

```diff
@@ -374,7 +402,7 @@ The Decimal type contains a scale, but no precision. The implied precision of a
 
 | Logical Type         | Physical Type               | Type ID | Equivalent Parquet Type     | Binary format                                                                                                       |
```

Comment:
@gene-db  When we say Logical Type and Physical Type here, what are we exactly referring to? Should we refer to Parquet logical type and Parquet physical type?

Reply:
Updated to clarify the columns that are Variant types.

For the Parquet types, I'd prefer to keep it as is to avoid a larger table. Even though it mixes logical and physical types, it is unambiguous.